### ISSUE TITLE
Feature req: add per provider option for interface to bind to.

### ISSUE THREAD

**Issue by nihilus** [troglobit/inadyn#246]
It would be nice to tell which interface that a specific provider should be queried/replied via.

**Comment by troglobit** [comment]
Yup. For now you'll have to run multiple instances instead.  I recommend using `--ident=provider` which will cause inadyn to look for /etc/provider.conf and create /run/provider.pid and log with the same identity to syslog.

**Comment by nihilus** [comment]
Actually I piggybacked on the proxy-functionality and used SO_BINDTODEVICE
